administration/src/components/AppToaster.tsx

```diff
@@ -0,0 +1,13 @@
+import {Position, Toaster} from "@blueprintjs/core";
+import {createRoot} from "react-dom/client";
+
+/** Singleton toaster instance. Create separate instances for different options. */
+let toasterRef: Toaster | null
+createRoot(document.getElementById("toaster")!).render(
+    <Toaster ref={ref => {
+        toasterRef = ref
+    }} position={Position.TOP}/>
+)
+export const getAppToaster = () => {
+    return toasterRef!
+}
```

Comment:
Oke this is weird. so the toaster has an own custom virtual dom? weird.

Reply:
I agree! I adjusted it to use the same DOM now. Maybe you can take another look.